### ISSUE TITLE
Storage calculation without other user's storage stats

### ISSUE THREAD

**Issue by samweisgamdschie** [arnovr/ocusagecharts#87]
Hi! After upgrade to 8.1.1 I installed charts app for the first time. While the activity stats looks correct the storage calculations (all three) are from a my own user (in admin group) and free space, no other user stats are available in this view.

OS: CentOS release 6.7 (Final)
PHP: PHP 5.5.28 (cli) (built: Aug  6 2015 08:53:05) 
OC: 8.1.1

Help to solve this problem is appreciated.

**Comment by samweisgamdschie** [comment]
Hi! Do you have a solution for me?

**Comment by dkebler** [comment]
Seem to be having a similar problem.  
Running 8.1.4. (14.04 Ubuntu, latest php)
cron job good, running
ran  below from command line
```` ./occ ocusagecharts:createdefaultcharts ````
but only a single "admin" user's usage shows up, yet activity charts show all users

any information I can provide that would be helpful?

**Comment by arnovr** [comment]
Sorry for the very very late reaction, i just had a kid.

The activity chart derives it information from the activity app. The storagechart doesn't. The createdefaultcharts should fill all system users. It retrieved all users based on this call:

    \OC_User::getUsers();

They might broke backwards compatibility in the latest versions?

You have 3 options:
1) Fix it for me? :)
2) Login as the user you want to track, click charts app, and it should be tracked.
3) Insert a record for each user into uc_chartconfig table

**Comment by samweisgamdschie** [comment]
for the third option, it seems that there is more than one line per user. Is it necessary to create each of these lines? 

mysql> select \* from oc_uc_chartconfig;
+----+---------------------+--------------+------------------------+---------------+---------------+
| id | created             | username     | charttype              | chartprovider | metadata      |
+----+---------------------+--------------+------------------------+---------------+---------------+
|  1 | 2015-09-01 07:51:50 | xxx        | StorageUsageCurrent    | c3js          |               |
|  2 | 2015-09-01 07:51:50 | xxx        | StorageUsageLastMonth  | c3js          | {"size":"gb"} |
|  3 | 2015-09-01 07:51:50 | xxx        | StorageUsagePerMonth   | c3js          | {"size":"gb"} |
|  4 | 2015-09-01 07:51:50 | xxx        | ActivityUsagePerMonth  | c3js          |               |
|  5 | 2015-09-01 07:51:50 | xxx        | ActivityUsageLastMonth | c3js          |               |

**Comment by arnovr** [comment]
Yes, for each user there are 5 charts. Each line represents 1 chart.
Skip a line, it will not show that chart.

In the new version of ocusagecharts this chartconfig stuff will be removed anyways.
